bitrise.yml

```diff
@@ -91,7 +91,7 @@ workflows:
     - certificate-and-profile-installer:
     - path::./:
         inputs:
-        - output_tool: xcodebuild
+        - output_tool: $OUTPUT_TOOL
```

Comment:
OUTPUT_TOOL is set to xcpretty (which is the default value for this input) please wire in test for xcodebuild output tool

Reply:
The test_objc workflow using the `xcodebuild` as an output tool.